go/interpreter/lfvm/instructions_test.go

```diff
@@ -870,3 +873,189 @@ func TestExpansionCostOverflow(t *testing.T) {
+		expectedStatus Status
+	}{
+		"berlin-enough-gas": {
+			runtimeSetup: func(runContext *tosca.MockRunContext) {
+				runContext.EXPECT().IsSlotInAccessList(gomock.Any(), gomock.Any()).Return(true, true).AnyTimes()
+			},
+			revision:       tosca.R09_Berlin,
+			gas:            SstoreSetGasEIP2200,
+			expectedStatus: RUNNING,
+		},
+		"berlin-not-enough-gas": {
+			runtimeSetup: func(runContext *tosca.MockRunContext) {
+				runContext.EXPECT().IsSlotInAccessList(gomock.Any(), gomock.Any()).Return(true, true).AnyTimes()
+			},
+			revision:       tosca.R09_Berlin,
+			gas:            SstoreSetGasEIP2200 - 1,
+			expectedStatus: OUT_OF_GAS,
+		},
+		"london-enough-gas": {
+			runtimeSetup: func(runContext *tosca.MockRunContext) {
+				runContext.EXPECT().IsSlotInAccessList(gomock.Any(), gomock.Any()).Return(true, true).AnyTimes()
+			},
+			revision:       tosca.R10_London,
+			gas:            SstoreSetGasEIP2200,
+			expectedStatus: RUNNING,
+		},
+		"london-not-enough-gas": {
+			runtimeSetup: func(runContext *tosca.MockRunContext) {
+				runContext.EXPECT().IsSlotInAccessList(gomock.Any(), gomock.Any()).Return(true, true).AnyTimes()
+			},
+			revision:       tosca.R10_London,
+			gas:            SstoreSetGasEIP2200 - 1,
+			expectedStatus: OUT_OF_GAS,
+		},
+		"minimum-gas-invariant": {
+			runtimeSetup: func(runContext *tosca.MockRunContext) {
+				runContext.EXPECT().IsSlotInAccessList(gomock.Any(), gomock.Any()).Return(true, true).AnyTimes()
+			},
+			revision:       tosca.R09_Berlin,
+			gas:            SstoreSentryGasEIP2200,
+			expectedStatus: OUT_OF_GAS,
+		},
+		"address-not-present": {
+			runtimeSetup: func(runContext *tosca.MockRunContext) {
+				runContext.EXPECT().IsSlotInAccessList(gomock.Any(), gomock.Any()).Return(false, false).AnyTimes()
+			},
+			revision:       tosca.R09_Berlin,
+			gas:            SstoreSetGasEIP2200,
+			expectedStatus: ERROR,
+		},
+	}
+
+	for name, test := range tests {
+		t.Run(name, func(t *testing.T) {
+			ctrl := gomock.NewController(t)
+			runContext := tosca.NewMockRunContext(ctrl)
+			test.runtimeSetup(runContext)
+
+			ctxt := context{
+				status:  RUNNING,
+				stack:   NewStack(),
+				memory:  NewMemory(),
+				context: runContext,
+				gas:     test.gas,
+			}
+			ctxt.revision = test.revision
+
+			// Prepare stack arguments.
+			ctxt.stack.push(uint256.NewInt(1))
+			ctxt.stack.push(uint256.NewInt(1))
+
+			runContext.EXPECT().SetStorage(gomock.Any(), gomock.Any(), gomock.Any()).Return(tosca.StorageAssigned).AnyTimes()
+			runContext.EXPECT().GetStorage(gomock.Any(), gomock.Any()).Return(tosca.Word{}).AnyTimes()
+			runContext.EXPECT().GetCommittedStorage(gomock.Any(), gomock.Any()).Return(tosca.Word{}).AnyTimes()
+
+			opSstore(&ctxt)
+
+			if ctxt.status != test.expectedStatus {
+				t.Errorf("unexpected status after call, wanted %v, got %v", test.expectedStatus, ctxt.status)
+			}
+		})
+	}
+}
+
+func TestSelfDestruct(t *testing.T) {
+
+	tests := map[string]struct {
```

Comment:
Similar to the other test, the `AnyTimes()` can be removed for all cases by removing the following lines:
`berlin-regular`: `GetBalance`
`london-address-not-in-list`: `HasSelfDestructed`
`london-create-new-account`: `HasSelfDestructed`